okhttp/src/jvmMain/kotlin/okhttp3/Cookie.kt

```diff
@@ -326,25 +348,38 @@ class Cookie private constructor(
       this.httpOnly = true
     }
 
+    fun samesite(sameSite: String) = apply {
+      val validSameSites = listOf("lax", "strict", "none")
+      require(validSameSites.contains(sameSite.lowercase())) { "SameSite must be one of: ${validSameSites.joinToString(", ")}" }
```

Comment:
We can’t be this strict here. New values may come before OkHttp is updated.

Reply:
What is the recommendation? Don't validate what's passed in, or add a warning log that says the value passed in is not a recognized value? For parsing, I think it makes sense to parse whatever fields people set even if it breaks the rules. Do we apply the same philosophy for cookie construction?